client/src/helpers/api.ts

```diff
@@ -359,19 +361,16 @@ const updateServiceRequestStatus = async (
     return ext;
   });
 
-  let notes: any[] = get(serviceRequest, 'note', []).filter((n) => n.text != null);
+  const notes: any[] = get(serviceRequest, 'note', [...DEFAULT_NOTES]).filter((n: Note) => n.text != null);
 
   if (note && note.length > 0) {
-    notes = [
-      ...notes,
-      {
-        authorReference: {
-          reference: `Practitioner/${user.practitionerData.practitioner.id}`,
-        },
-        text: note,
-        time: new Date(),
+    updateNoteStatus(notes, {
+      authorReference: {
+        reference: `Practitioner/${user.practitionerData.practitioner.id}`,
       },
-    ];
+      text: note,
+      time: new Date(),
```

Comment:
Question: should we use **UTC**?

Reply:
Seems to be UTC but can't find confirmation on the documentation ... will check that

Reply:
Alright, I candidly have no idea. Interested to know about it. So update me, pls.

Reply:
`new Date().toISOString()` is the solution, it returns UTC formated like `2011-10-05T14:48:00.000Z`